config/routes.rb

```diff
@@ -8,4 +9,12 @@
       resources :likes, only: %i[new create]
     end
   end
+
+  namespace :api, default: {format: :json} do
+    namespace :v1 do
+      resources :posts, only: %i[index] do
+        resources :comments, only: %i[index create]
+      end
+    end
```

Comment:
[REQUIRED] Kindly ensure that posts and comments resources are wrapped by users. So comments are nested in post as you have done and posts are nested in users. If you notice you are asked to create endpoints for posts and comments that belong to a user. Nesting it like that will give use endpoints that look something like this `/api/v1/users/:user_id/posts...`